### DISCUSSION TITLE
Usage of gameplay keys in the pause menu

### DISCUSSION THREAD

**Discussion by Sprinter05** [ppy/osu#21881] [Ideas] [answered]
In stable's pause menu you can use the gameplay keys (e.g. Z X or whatever keys the user has set) to click the different options the same way you can use the mouse buttons. I miss this feature from stable because I always click "Retry" on the pause menu using those keys and that way i dont need to move my hand to the "Quick restart" key or the "Enter"/"Left mouse click" key. It would be nice to have this back.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/3551.
